indico/modules/events/papers/templates/_contributions.html

```diff
@@ -166,3 +166,71 @@ <h2>{% trans %}Editing{% endtrans %}</h2>
         </div>
     {% endif %}
 {% endmacro %}
+
+{% macro _render_file_type_section(latest_revision_files, file_type) %}
+    <div class="files field-separator">
+        <div class="attachments-box">
+            <div class="attachments-box-header">
+                <h3 class="file-type-header">{{ file_type.name }}</h3>
+            </div>
+            <table class="tree">
+                {% for rev_file in latest_revision_files %}
+                    {% set file = rev_file.file %}
+                    {% set download_url = rev_file.download_url %}
+                    {% set icon = icon_from_mimetype(file.content_type, 'icon-file-filled') %}
+                    <tr>
+                        <td class="revision-file-row">
+                            <a href="{{ download_url }}" class="attachment {{ icon }}" target="_blank">
+                                {{ file.filename }}
+                            </a>
+                        </td>
+                    </tr>
+                {% endfor %}
+            </table>
+        </div>
+    </div>
+{% endmacro %}
+
+
+{% macro render_proceedings_section(contribution) %}
+    {% set enabled_editables = contribution.enabled_editables %}
+    <section>
+        <div class="header">
+            <div class="header-row">
+                <h2>{% trans %}Proceedings{% endtrans %}</h2>
+            </div>
+        </div>
+        {% for editable in enabled_editables -%}
```

Comment:
{% for editable in enabled_editables if editable.published_revision -%}

and use `editable.published_revision` below instead of `latest_revision`; the state check can go away as well